src/reformat_libyuv.c

```diff
@@ -341,6 +341,38 @@ avifResult avifImageYUVToRGBLibYUV(const avifImage * image, avifRGBImage * rgb)
     return AVIF_RESULT_NOT_IMPLEMENTED;
 }
 
+// These defines are used to create a NULL reference to libyuv functions that
+// did not exist prior to a particular version of libyuv.
+#if LIBYUV_VERSION < 1838
+#define I422ToRGB565Matrix NULL
+#endif
+#if LIBYUV_VERSION < 1813
+#define I422ToARGBMatrixFilter NULL
+#define I420ToARGBMatrixFilter NULL
+#define I210ToARGBMatrixFilter NULL
+#define I010ToARGBMatrixFilter NULL
+#endif
+#if LIBYUV_VERSION < 1780
+#define I410ToARGBMatrix NULL
+#endif
+#if LIBYUV_VERSION < 1756
+#define I400ToARGBMatrix NULL
+#endif
+
+#define RGB_PIXEL_FORMATS 7 // Number of enum values in avifRGBFormat.
+#define YUV_PIXEL_FORMATS 5 // Number of enum values in avifPixelFormat.
+
+// Look up table for isYVU.
```

Comment:
Nit: It would be good to define `isYVU`. It means we are using a libyuv function with R and B swaped, which requires U and V also be swapped.